gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -137,13 +145,17 @@
 
   private final Map<MapFeature, List<LatLng>> geoJsonPolygonLoops = new HashMap<>();
   private final Map<MapFeature, ArrayList<ArrayList<LatLng>>> geoJsonPolygonHoles = new HashMap<>();
+  private final Map<MapFeature, List<LatLng>> polygons = new HashMap<>();
 
   private int cameraChangeReason = REASON_DEVELOPER_ANIMATION;
 
-  public GoogleMapsMapAdapter(GoogleMap map, Context context, MarkerIconFactory markerIconFactory) {
+  public GoogleMapsMapAdapter(GoogleMap map, Context context,
+      MarkerIconFactory markerIconFactory,
+      BitmapUtil bitmapUtil) {
     this.map = map;
     this.context = context;
     this.markerIconFactory = markerIconFactory;
+    this.customCap = new CustomCap(bitmapUtil.bitmapDescriptorFromVector());
```

Comment:
From which vector? I wouldn't expect BitmapUtil to have which vector to convert hard-coded. Perhaps you want to pass it as an arg here?